src/components/EditCoursePage/index.jsx

```diff
@@ -126,19 +125,31 @@ class EditCoursePage extends React.Component {
     const {
       editCourse,
     } = this.props;
-    // Create a deep copy of course runs since we modify their properties
-    let modifiedCourseRuns = jsonDeepCopy(courseData.course_runs);
 
-    // Don't send any courses in review - backend will reject them
-    modifiedCourseRuns = modifiedCourseRuns.filter(run => !IN_REVIEW_STATUS.includes(run.status));
+    const { targetRun } = this.state;
 
-    // Process course run info
-    this.prepareCourseRuns(modifiedCourseRuns, courseData);
+    // Process course run info from courseData
+    const modifiedCourseRuns = this.prepareCourseRuns(courseData);
+    // Update the Draft status based on run status
+    let courseEditData = jsonDeepCopy(courseData); // eslint-disable-line prefer-const
```

Comment:
Thinking about it more, maybe we let `prepareCourseRuns` and `prepareCourse` modify their input data since they are merely helper functions to `handleCourseEdit`. Then we can just move this line to the top of the function and remove the other deepcopys.

I realized we are deep copying the course runs in prepareCourseRuns, and then we technically deep copy them again when we deep copy the entire courseData in prepareCourse so why not just deep copy it once in the beginning and then when prepareCourseRuns needs course_runs, it can have it from the object that has already been copied.